events/SignIn/GetSigninToken.json

```diff
@@ -9,6 +9,19 @@
     "mitreAttackTechniques": [
         "T1078 - Valid Accounts"
     ],
+    "mitreAttackSubTechniques": [
+
+    ],
+    "unverifiedMitreAttackTechniques": [
+        {
+            "technique": "T1562 - Impair Defenses",
+            "reason": "Access via the SigninToken enables an attacker to alter or disable cloud defenses, such as modifying IAM policies, Security Groups, or other security mechanisms to evade detection."
+        },
+        {
+            "technique": "T1589 - Gather Victim Network Information",
+            "reason": "Using the AWS Management Console accessed via the token, an attacker could collect information about the network's architecture, VPC configurations, and connected services."
```

Comment:
I think these 2 techniques are due not to GetSigninToken, but after getting a valid token with permissions, in this case you can execute many other techniques.

Reply:
I'll remove both.